src/06-loops.js

```diff
@@ -25,7 +32,17 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
```

Comment:
It's usually a good idea to iterate forward through an array unless we need to do otherwise.  While this code works the convention is to increase our index as we move through an array